caddyhttp/fastcgi/fastcgi.go

```diff
@@ -10,12 +10,18 @@ import (
 	"os"
 	"path"
 	"path/filepath"
+	"runtime"
 	"strconv"
 	"strings"
+	"sync"
 
 	"github.com/mholt/caddy/caddyhttp/httpserver"
 )
 
+// for persistent fastcgi connections
+var persistentConnections map[string]*FCGIClient
+var poorMansSerialisation map[string]*sync.Mutex
```

Comment:
Use `make()` for both of these here so you don't have to have extra logic below (see comment)